### PR TITLE
docs(lettable-operator): recommend ts-import-plugin for tree-shaking

### PR DESCRIPTION
**Pull request by Brooooooklyn**

### CODE REVIEW

**Comment by kwonoj** [comment]
I'm not sure we'd like to encourage specific plugin in official readme. /cc @jasonaden for visibility and assigning @benlesh for review, but personally downvote for this.

File: doc/lettable-operators.md [hunk 1]
```diff
@@ -219,6 +219,65 @@ var config = {
 module.exports = config;
 ```
 
+**Use ts-import-plugin**
```
Comment (jasonaden) [inline doc/lettable-operators.md]: Add another line break after this.

File: doc/lettable-operators.md [hunk 2]
```diff
@@ -219,6 +219,65 @@ var config = {
...
+You can use [ts-import-plugin](https://www.npmjs.com/package/ts-import-plugin) to transform your import clause with ts-loader.
+For example, it can transform
```
Comment (jasonaden) [inline doc/lettable-operators.md]: And another line break here.

**Comment by kwonoj** [comment]
My concern here is not avout usefulness of plugin but doc implies it as sort of de facto recommended in official repo. For same reason we haven't added other 3rd party repo in doc links.

For me, this is more feasible for rxjs-docs' awesome-list we are trying to list modules for ecosystem.

**Comment by benlesh** [comment]
Yeah. I think this is something where it would be good to have some blog posts on approaches for tree-shaking that we can recommend in some sort of 'awesome list' or the like. But I don't want us to have to maintain an up-to-date list of the best approaches to tree-shaking. We have enough on our plate.

I agree with @kwonoj. We should probably close this one.

**Comment by Brooooooklyn** [comment]
oh thanks，maybe I will write a blog about this later😄

**Pull request closed without merging**